src/webgpu/shader/execution/expression/binary/f32_matrix_arithmetic.spec.ts

```diff
@@ -170,6 +171,438 @@ export const d = makeCaseCache('binary/f32_matrix_arithmetic', {
       additionMatrixInterval
     );
   },
+  multiplication_2x2_2x2_const: () => {
```

Comment:
Blimey, do we know how many cases this is in total?

Reply:
Just checked, 17004, and that is using sparseMatrixRange.

Reply:
Ouch. We'll have to check just how long this adds to the CTS.